src/router/index.js

```diff
@@ -453,42 +488,16 @@ router.beforeEach(async (to, from, next) => {
     return;
   }
 
-  // Check if the route requires admin rights and the user is an admin.
-  const requiresAdmin = _get(to, 'meta.requireAdmin', false);
-  const requiresSuperAdmin = _get(to, 'meta.requireSuperAdmin', false);
-
-  // Check user roles
-  const isUserAdmin = store.isUserAdmin;
-  const isUserSuperAdmin = store.isUserSuperAdmin;
-
-  // All current conditions:
-  // 1. Super Admin: true, Admin: true
-  // 2. Super Admin: false, Admin: true (Only exits because requiresSuperAdmin is not defined on every route)
-  // 3. Super Admin: false, Admin: false (Allowed routes for all users)
-  // (Also exists because requiresAdmin/requiresSuperAdmin is not defined on every route)
-
-  if (isUserSuperAdmin) {
-    next();
-    return;
-  } else if (isUserAdmin) {
-    // LEVANTE dashboard has opened some pages to administrators before the ROAR dashboard
-    // So if isLevante, then allow regular admins to access any route with requireAdmin = true.
-    // and if ROAR, then prohibit regular admins from accessing any route with requireSuperAdmin = true.
-    if (isLevante && requiresAdmin) {
+  const routePermission = _get(to, 'meta.permission', null);
```

Comment:
No need to import `lodash` for this: 

```suggestion
  const routePermission = to?.meta?.permission ?? null
```

A good ressource for these sort of things: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore